compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -1362,6 +1366,8 @@ impl<'ra, 'tcx> Resolver<'ra, 'tcx> {
         );
 
         if lookup_ident.span.at_least_rust_2018() {
+            // Extended suggestions will be sorted at the end of this function.
+            #[allow(rustc::potential_query_instability)]
             for ident in self.extern_prelude.clone().into_keys() {
```

Comment:
Could you sort the extern prelude here instead of the suggestions at the bottom of the function?

Reply:
Done: https://github.com/rust-lang/rust/pull/131213/commits/0944d8c1ea3bdcc40b14605a4c884acfb0f50853